suites/reef/cephadm/tier-2-prometheus-retention-time.yaml

```diff
@@ -0,0 +1,96 @@
+#===============================================================================================
+# Test case: Verify Prometheus daemon retention time is adjustable
+#
+# Conf: conf/reef/cephadm/tier-0.yaml
+#===============================================================================================
+tests:
```

Comment:
Why separate test suite ? Can't it be part of any existing test suite ?

Reply:
This test case is part of cephadm component and there is no other tier-2 test suite where I can add this Prometheus related test case.